loader/loader.go

```diff
@@ -220,10 +226,10 @@ func (l *loader) GetRelease(ctx context.Context, cli client.Client, name, namesp
 // by the namespaced name stored in the Release's status.
 func (l *loader) GetRoleBindingFromReleaseStatus(ctx context.Context, cli client.Client, release *v1alpha1.Release) (*rbac.RoleBinding, error) {
 	roleBinding := &rbac.RoleBinding{}
-	roleBindingNamespacedName := strings.Split(release.Status.Processing.RoleBinding, string(types.Separator))
+	roleBindingNamespacedName := strings.Split(release.Status.ManagedProcessing.RoleBinding, string(types.Separator))
```

Comment:
I'm a bit confused. The RoleBinding only makes sense in the context of the Managed Pipeline but now as we reuse the PipelineInfo, every single pipeline has that field. Should we move it to a different place? We have 4 pipeline types and only one requires that field.

Reply:
I am fine with moving it, but do you have any ideas of how to do it cleanly? It would kind of suck to have a `PipelineInfoWithRolebinding` struct but that is kind of the cleanest way I can think to do it

Reply:
Let's keep it this way and maybe go for a general clean up of the service code when we run up of tickets? Pretty sure we can improve some things now that we have more experience.